velox/docs/functions/spark/aggregate.rst

```diff
@@ -11,6 +11,37 @@ General Aggregate Functions
 
     Returns the bitwise XOR of all non-null input values, or null if none.
 
+.. spark:function:: bloom_filter_agg(hash, estimatedNumItems, numBits) -> varbinary
+
+    Creates bloom filter from input hashes and returns it serialized into VARBINARY.
+    The caller is expected to apply xxhash64 function to input data before calling bloom_filter_agg.
+    For example, 
+        bloom_filter_agg(xxhash64(x), 100, 1024)   
```

Comment:
Would you generate the docs locally and verify they get formatted nicely. It seems to me that we need some new lines or something around the example.

Reply:
Sorry for the document, I don't know how to confirm the document before.
Now I know how to convert it to html and use Vscode to preview, I will check the html format. Thanks for your kindly review.